frontend/src/features/archived-items/crawl-status.ts

```diff
@@ -155,11 +158,15 @@ export class CrawlStatus extends LitElement {
 
       case "complete":
         icon = html`<sl-icon
-          name=${isUpload ? "upload" : "check-circle-fill"}
+          name=${type === "upload" ? "upload" : "check-circle-fill"}
           slot="prefix"
           style="color: var(--success)"
         ></sl-icon>`;
-        label = isUpload ? msg("Uploaded") : msg("Complete");
+        label = {
+          upload: msg("Uploaded"),
+          crawl: msg("Complete"),
+          qa: msg("Ready for Review"),
```

Comment:
I believe you were out when we discussed this, sorry that notes are missing, but we're decoupling the QA analysis and QA review statuses, so this can just be "complete".
```suggestion
          qa: msg("Complete"),
```

Reply:
Ah cool, thanks! Were there any other changes from the mockups I should be aware of? cc @Shrinks99